programs/steward/src/state/steward_state.rs

```diff
@@ -447,7 +471,26 @@ impl StewardState {
         self.delegations[self.num_pool_validators] = Delegation::default();
         self.instant_unstake.set(self.num_pool_validators, false)?;
         self.progress.set(self.num_pool_validators, false)?;
+        self.progress.set(self.num_pool_validators, false)?;
```

Comment:
duplicate line

Reply:
nice catch - should be `num_pool_validators`